modules/epang/meta.yml

```diff
@@ -0,0 +1,67 @@
+name: "epang"
+description: phylogenetic placement of query sequences in a reference tree
+keywords:
+  - phylogeny
+  - phylogenetic placement
+  - sequences
+tools:
+  - "epang":
+      description: "Massively parallel phylogenetic placement of genetic sequences"
+      homepage: "https://github.com/Pbdas/epa-ng"
+      documentation: "https://github.com/Pbdas/epa-ng/wiki/Full-Stack-Example"
+      tool_dev_url: "https://github.com/Pbdas/epa-ng"
+      doi: "https://doi.org/10.1093/sysbio/syy054"
+      licence: "['GNU Affero General Public License v3.0']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', model:'LG' ]
+  - queryaln:
+      type: file
+      description: aligned query sequences in any supported format including phylip and fasta, may be gzipped
+      pattern: "*"
```

Comment:
```suggestion
      pattern: "*.{fa,fas,fna,fasta,fa.gz,.fas.gz,fna.gz,fasta.gz,.phylip,.phylip.gz}"
```

Reply:
There's `faa` and `faa.gz` too, plus I'm using `alnfaa`/`alnfna` for aligned fasta, I'm sure there are other suffixes. But I suppose this is not validated anywhere, or? If so, one could of course have these patterns, but I don't see the point really if the list is very long.

Reply:
Ok fair enough! Can ignore this one